deploy/docker/Dockerfile

```diff
@@ -77,4 +77,4 @@ EXPOSE 1883 8081 8083 8084 8883 11883 18083 4369 4370 5369 6369
 
 ENTRYPOINT ["/usr/bin/docker-entrypoint.sh"]
 
-CMD ["/usr/bin/start.sh"]
+CMD ["/opt/emqx/bin/emqx", "foreground"]
```

Comment:
Please delete `EMQX_WAIT_TIME` in `https://github.com/emqx/emqx/blob/dev/v4.3.0/deploy/docker/README.md`

Reply:
fixed in a fixup